heron/packing/tests/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPackingTest.java

```diff
@@ -97,16 +101,18 @@ private static PackingPlan doScalingTest(TopologyAPI.Topology topology,
 
     PackingPlan newPackingPlan =
         getFirstFitDecreasingPackingPlanRepack(topology, packingPlan, componentChanges);
-    return newPackingPlan;
+    packingInfo = new Pair<PackingPlan, Resource>(
+        newPackingPlan, packingPlan.getMaxContainerResources());
```

Comment:
Why is Resouce `packingPlan.getMaxContainerResources()` and not `newPackingPlan.getMaxContainerResources()`? What does it represent?